File: Exam/IssueTracker/JavaScript/models/issue.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Issue = sequelize.define("Issue",{
        title: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },

        content: {
            type: Sequelize.TEXT,
            required: true,
            allowNull: false
        },

        priority: {
            type: Sequelize.INTEGER,
            required: true,
            allowNull: false
        }
    },{
        timestamps:false
    });

    return Issue;
};

File: Exam/IssueTracker/C#/IssueTracker/Models/Issue.cs
﻿namespace IssueTracker.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Issue
    {
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Content { get; set; }

        [Required]
        public int Priority { get; set; }
    }
}
